packages/harmony/src/components/expandable-nav-item/ExpandableNavItem.tsx

```diff
@@ -27,20 +31,14 @@ const variants: VariantConfigs = {
   }
 }
 
-const getStyles = (theme: HarmonyTheme, isHovered: boolean): CSSObject => {
-  const baseStyles: CSSObject = {
-    transition: `background-color ${theme.motion.hover}`,
-    cursor: 'pointer'
-  }
-
-  const hoverStyles: CSSObject = {
-    backgroundColor: theme.color.background.surface2,
-    boxShadow: `inset 0 0 0 1px ${theme.color.border.default}`
-  }
-
+const getStyles = (
+  theme: HarmonyTheme,
+  isHovered: boolean,
```

Comment:
this is unused now

Reply:
don't really need these two to be in the function anymore, just put with the rest of the styles below